bokeh/properties.py

```diff
@@ -837,7 +837,8 @@ class List(Seq):
 
     """
 
-    def __init__(self, item_type, default=[], help=None):
+    def __init__(self, item_type, default=None, help=None):
+        default = default or []
```

Comment:
We want to support both `None` and `[]`, because we currently don't have better way to express optional values (I wanted to implement things like `Option(List(Int))`, but haven't find time for this). If you do `default or []`, then:
```
>>> None or []
[]
```
and is what we don't want.